smoke-test/src/test/java/org/opennms/smoketest/GrafanaEndpointPageIT.java

```diff
@@ -99,17 +135,43 @@ public void verifyCRUD() {
     }
 
     @Test
-    public void verifyTestConnection() {
-        final GrafanaEndpoint dummyEndpoint = GrafanaEndpointRestIT.createDummyEndpoint();
+    public void verifyTestConnection() throws JsonProcessingException {
```

Comment:
Instead of replacing this, write a new one to test a real connection.

Reply:
Resolved.reverted this .